src/test/Transformations/shaders/Triangle/src/2DTransformationTest.vert.glsl

```diff
@@ -2,11 +2,12 @@
 
 layout(location = 0) in vec3 inPos;
 
-layout(location = 100) uniform mat4 ModelMatrix;
-layout(location = 101) uniform mat4 ViewMatrix;
-layout(location = 102) uniform mat4 ProjectionMatrix;
+//layout(location = 100) uniform mat4 ModelMatrix;
+//layout(location = 101) uniform mat4 ViewMatrix;
+//layout(location = 102) uniform mat4 ProjectionMatrix;
```

Comment:
Restore this file (and compile it to get the corresponding binary) to what it was.